src/debugProtocol/responses/ProtocolEvent.spec.ts

```diff
@@ -0,0 +1,71 @@
+import { ProtocolEvent } from './ProtocolEvent';
+import { createProtocolEvent } from './responseCreationHelpers.spec';
+import { expect } from 'chai';
+import { createSandbox } from 'sinon';
+import { ERROR_CODES, UPDATE_TYPES } from '../Constants';
+const sinon = createSandbox();
+
+describe('ProtocolEvent', () => {
+    it('Handles a Protocol update events', () => {
+        let mockResponse = createProtocolEvent({
+            requestId: 0,
+            errorCode: ERROR_CODES.CANT_CONTINUE,
+            updateType: UPDATE_TYPES.ALL_THREADS_STOPPED
+        });
+
+        let protocolEvent = new ProtocolEvent(mockResponse.toBuffer());
+        expect(protocolEvent.requestId).to.be.equal(0);
+        expect(protocolEvent.errorCode).to.be.equal(ERROR_CODES.CANT_CONTINUE);
+        expect(protocolEvent.updateType).to.be.equal(UPDATE_TYPES.ALL_THREADS_STOPPED);
+        expect(protocolEvent.readOffset).to.be.equal(mockResponse.writeOffset);
+        expect(protocolEvent.success).to.be.equal(true);
+    });
+
+    it('Handles a Protocol response events', () => {
+        let mockResponse = createProtocolEvent({
+            requestId: 1,
+            errorCode: ERROR_CODES.OK
+        });
+
+        let protocolEvent = new ProtocolEvent(mockResponse.toBuffer());
+        expect(protocolEvent.requestId).to.be.equal(1);
+        expect(protocolEvent.errorCode).to.be.equal(ERROR_CODES.OK);
+        expect(protocolEvent.updateType).to.be.equal(-1);
+        expect(protocolEvent.readOffset).to.be.equal(mockResponse.writeOffset);
+        expect(protocolEvent.success).to.be.equal(true);
+    });
+
+    // it('Fails when buffer is incomplete', () => {
```

Comment:
Do these need to be uncommented and fixed? If not, please remove them or add a comment as to why they're still here.

Reply:
I'll check and update

Reply:
Updated the tests.